src/main/java/com/cdancy/bitbucket/rest/options/UpdateTask.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cdancy.bitbucket.rest.options;
+
+import com.cdancy.bitbucket.rest.domain.comment.TaskAnchor;
+import com.google.auto.value.AutoValue;
+import org.jclouds.json.SerializedNames;
+
+@AutoValue
+public abstract class UpdateTask {
+
+    public abstract TaskAnchor anchor();
+
+    public abstract int id();
+
+    public abstract String state();
+
+    public abstract int pullRequestId();
+
+    public abstract int repositoryId();
+
+
+    UpdateTask() {
+    }
+
+    @SerializedNames({"anchor", "id", "state", "pullRequestId", "repositoryId" })
+    public static UpdateTask update(final TaskAnchor anchor,
```

Comment:
lets change the method name to `create`. I'm fine with the class name being different however.

Reply:
Will do

Reply:
Do we still need anchor? The API suggests we don't.

Reply:
If we get rid of anchor I have a way where we will be able to get rid of the `UpdateTask` altogether and have the user just pass along string/int properties which would simplify things but ONLY if we _can_ get rid of anchor.

Reply:
After some pruning down of extra API details I got to the point where I see what you mean. So, in 007664f, I removed the `UpdateTask` class and just modified the TasksApi to accept a string representing the state. Followed up with modifying the tests to reflect this. 

I think this is a much nicer API to call although having the client code ensure it sends the correct strings is not ideal (but not sure what we can do about it since we're building on an existing API). Ideally this could just be a toggle method accepting a boolean for setting a task to opened or resolved.

Reply:
Also, the reason I had made things more complicated with regard to request payloads is because I used Chrome developer tools to see what payloads were sent when using the Web UI. I _should_ have just read the fine docs to see what was really required versus extra stuff :)